packages/falso/src/lib/text.ts

```diff
@@ -1,6 +1,67 @@
-import { FakeOptions, fake } from './core';
-import { data } from './text.json';
+import { FakeOptions, fake, getRandomInRange } from './core';
+import { word } from './word';
 
-export function text<Options extends FakeOptions>(options?: Options) {
-  return fake(data, options);
+export interface TextOptions extends FakeOptions {
+  charCount?: number;
+}
+
+function getSpecialCharacter(): string {
+  const randomNumber = getRandomInRange({ min: 1, max: 10, fraction: 0 });
+
+  if (randomNumber === 1) {
+    return '.';
+  }
+
+  if (randomNumber === 2) {
+    return ',';
+  }
+
+  return '';
+}
+
+function generateSentence(charCount: number): string {
+  let text = word({ capitalise: true });
+  let capitalise = false;
+  let totalWords = 1;
+
+  while (text.length < charCount) {
+    let randomWord: string;
+    let specialChar = '';
+
+    if (capitalise) {
+      randomWord = word({ capitalise: true });
+      capitalise = false;
+    } else {
+      randomWord = word();
+    }
+
+    if (totalWords > 2) {
+      specialChar = getSpecialCharacter();
+    }
+
+    if (specialChar === '.') {
+      capitalise = true;
+    }
+
+    text += ` ${randomWord}${specialChar}`;
+    totalWords++;
+  }
+
+  return text.substring(0, charCount);
+}
+
+export function text<Options extends TextOptions>(
+  options?: Options
+): string | string[] | void {
+  const charCount: number = options?.charCount ?? 10;
+
+  const factory = () => {
+    if (charCount < 1 || isNaN(charCount)) {
```

Comment:
Do it once outside the factory.

Reply:
That's actually a good point. What is the difference between sentence and text? Is there a need for both?

Reply:
That's a good point. What is the difference between sentence and text? Is there a need for both?

If I move my sentence building logic into `sentence.ts`, what will be the difference between the 2?